lib/puppet/provider/apt_key/apt_key.rb

```diff
@@ -205,6 +205,9 @@ def create
     elsif resource[:source]
       key_file = source_to_file(resource[:source])
       command.push('add', key_file.path)
+      unless resource[:options].nil?
```

Comment:
I think we should have some acceptance tests here. `source` also supports absolute paths to local files and I'm not sure if gpg supports a keyserver option + local file to import.